heareval/tasks/config/speech_commands.py

```diff
@@ -18,21 +21,16 @@
 # want them all.
 SAMPLE_RATES = [48000, 44100, 22050, 16000]
 
-# TODO: Pick the 75th percentile length?
-SAMPLE_LENGTH_SECONDS = 8.0
+SAMPLE_LENGTH_SECONDS = 1.0
 
-# TODO: Do we want to call this FRAME_RATE or HOP_SIZE
-FRAME_RATE = 4
 
-# Set this to None if you want to use ALL the data.
-# NOTE: This will be, expected, 225 test files only :\
-# NOTE: You can make this smaller during development of this
-# preprocessing script, to keep the pipeline fast.
-# WARNING: If you change this value, you *must* delete _workdir
-# or working dir.
-# Most of the tasks iterate over every audio file present,
-# except for the one that downsamples the corpus.
-# (This is why we should have one working directory per task)
-MAX_FRAMES_PER_CORPUS = 20 * 3600
+MAX_HOURS = 10.0
 
-MAX_FILES_PER_CORPUS = int(MAX_FRAMES_PER_CORPUS / FRAME_RATE / SAMPLE_LENGTH_SECONDS)
+MAX_FILES_PER_CORPUS = int(MAX_HOURS * 60.0 * 60.0 / SAMPLE_LENGTH_SECONDS)
+
+# Number of test files to include - 4890 is the full test set
+MAX_TEST_FILES = 4890
```

Comment:
Do we want to include the entire test set? Might make sense to provide easy comparison

Reply:
Yeah why not